### ISSUE TITLE
Error when deleting a row

### ISSUE THREAD

**Issue by jimdodson** [tudor-malene/Easygrid#125]
Using Grails 2.4.3 and EasyGrid 1.6.8

When clicking the trash can icon on a row, I get the modal "delete selected record(s)?" dialog, but the clicked row also appears to be editable.  If I click Delete, the dialog shows "Internal Service Error" and the stack trace below is shown in the console. If I then refresh the page, the grid is reloaded and the row has been deleted.

Since the error occurred when trying to POST to the InlineEdit method, I'm wondering if somehow an edit is being attempted in addition to the delete? Or is my config wrong somehow?

Controller:

```
    def categoryGrid = {
        domainClass Category
        jqgrid {
            navGrid {
                generalOpts {
                    view false
                    search false
                }
            }
        }
        columns {
            actions {
                label ' '
                type "actions"
            }
            id {
                label "id"
                enableFilter false
                jqgrid {
                    editable "false"
                    resizable "false"
                    width 25
                }
            }
            // other columns
```

    Error |
    2014-11-04 14:08:01,988 [http-bio-8080-exec-9] ERROR errors.GrailsExceptionResolver  - NullPointerException occurred when processing request: [POST] /tmui/category/categoryInlineEdit - parameters:
    id: 23
    oper: del
    Cannot get property 'id' on null object. Stacktrace follows:
    Message: Cannot get property 'id' on null object
    Line | Method
    ->>  145 | transformInlineResponse in org.grails.plugin.easygrid.grids.JqueryGridService
    - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
    |     31 | methodMissing           in org.grails.plugin.easygrid.EasygridDispatchService
    |     89 | doCall . . . . . . . .  in org.grails.plugin.easygrid.EasygridInitService$_registerControllerMethods_closure6
    |    217 | guard                   in org.grails.plugin.easygrid.EasygridService
    |    110 | doCall . . . . . . . .  in org.grails.plugin.easygrid.EasygridInitService$_registerControllerMethods_closure9_closure17
    |    189 | doFilter                in grails.plugin.cache.web.filter.PageFragmentCachingFilter
    |     63 | doFilter . . . . . . .  in grails.plugin.cache.web.filter.AbstractFilter
    |     53 | doFilter                in grails.plugin.springsecurity.web.filter.GrailsAnonymousAuthenticationFilter
    |     49 | doFilter . . . . . . .  in grails.plugin.springsecurity.web.authentication.RequestHolderAuthenticationFilter
    |     82 | doFilter                in grails.plugin.springsecurity.web.authentication.logout.MutableLogoutFilter
    |   1145 | runWorker . . . . . . . in java.util.concurrent.ThreadPoolExecutor
    |    615 | run                     in java.util.concurrent.ThreadPoolExecutor$Worker
    ^    744 | run . . . . . . . . . . in java.lang.Thread

**Comment by amber2581** [comment]
Has there been any resolution to this issue? I just installed the newest version of easygrid and I am also having this issue everytime I delete a record in the grid.  Thanks!

**Comment by tudor-malene** [comment]
Should be fixed in 1.6.9.1

**Comment by amber2581** [comment]
The update 1.6.9.1 fixed this issue for me.  Thanks!
